### PR TITLE
LPD-15603 Get portalURL variable from http request

### PR DESCRIPTION
**Pull request by devpedrotavares**
Follow up on: https://github.com/brianchandotcom/liferay-portal/pull/146944

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/servlet/filter/ObjectActionContextFilter.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * @author Pedro Tavares
+ */
+@Component(
+	property = {
+		"servlet-context-name=",
+		"servlet-filter-name=Object Action Context Filter", "url-pattern=/o/c/*"
+	},
+	service = Filter.class
+)
+public class ObjectActionContextFilter extends BaseFilter {
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/servlet/filter/ObjectActionContextFilter.java:30]: As we discussed on Slack, rename this class to `ObjectDefinitionRESTContextPathFilter`

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by devpedrotavares** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:object

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:object

**Comment by devpedrotavares** [comment]
ci:report:2535862192

**Comment by devpedrotavares** [comment]
ci:report:2535869827

**Comment by devpedrotavares** [comment]
ci:forward:force

**Pull request closed without merging**